ui/views/tx/details/tx-status-view.js

```diff
@@ -24,39 +24,41 @@ function DateRowView({tooltip, children}) {
 export default function TxStatusView({tx}) {
     const {status, submitted, hash, network} = tx
     if (submitted) return <>
-        <StatusRowView
-            tooltip="The transaction has been signed, processed, and automatically submitted to Stellar network"
-            extraInfo={<a href={`https://stellar.expert/explorer/${network}/tx/${hash}`} target="_blank"
-                          className="icon-open-new-window" title="View in explorer"></a>}>
+        <StatusRowView tooltip="The transaction has been signed, processed, and automatically submitted to Stellar network"
+                       extraInfo={<a href={`https://stellar.expert/explorer/${network}/tx/${hash}`} target="_blank" rel="noreferrer"
+                                     className="icon-open-new-window" title="View in explorer"/>}>
             Executed
         </StatusRowView>
         <DateRowView>{submitted}</DateRowView>
     </>
     switch (status) {
         case 'pending':
-            return <StatusRowView tooltip="The transaction has not reached the required signatures threshold yet">
+            return <StatusRowView tooltip="The transaction has not reached the required signatures threshold yet"
+                                  extraInfo={<span className="loader small"/>}>
```

Comment:
```suggestion
            return <StatusRowView showLoader tooltip="The transaction has not reached the required signatures threshold yet">
```

It's better to introduce a boolean property `showLoader` and implement the loader display logic internally than copy-paste the same code in multiple rows.